Runtime/Patterns/ServiceLocator/ServiceLocator.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine.Assertions;
+
+namespace StansAssets.Foundation.Patterns
+{
+    /// <inheritdoc cref="IServiceLocator"/>
+    /// <remarks>
+    /// An implementation of the <see cref="IServiceLocator"/> pattern.
+    /// </remarks>
+    public class ServiceLocator : IServiceLocator
```

Comment:
Any point why we don't make this class `sealed`?
It doesn't have anything that might be overridden (at least at this moment), so let's maybe consider to make it `sealed`

Reply:
I would also recommend to start thinking about some abstract version of this class because each user who will want to write his own SL will just duplicate the code. But with abstract one it will be much more efficient/quick implement another SLs.

Reply:
I am not sure if we or somebody else would ever want to extend it.
Let's keep it `sealed` as @alex-yaremenko suggested.